### PR TITLE
Streamlining samples for multi-party collaboration using Azure Clean Rooms

### PR DESCRIPTION
**Pull request by PhaniChiruvolu**
…Room.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,731 @@
...
+```
+<br>
+
+<!-- TODO: Update block diagram to be more generic. -->
+<!-- Once the clean room is deployed the key components involved during execution are shown below:
+
+![alt text](./assets/encrypted-storage-layout.png) -->
+
+# Setting up the consortium
+Collaboration using a CCR is realized and governed through a consortium created in [CCF](https://microsoft.github.io/CCF/main/overview/what_is_ccf.html) hosting a [Clean Room Governance Service (CGS)](../../src/governance/README.md). All the collaborating parties become participating members in the consortium.
```
Comment (anantshankar17) [inline README.md:281]: >../../src/governance/README.md [](http://example.com/codeflow?start=201&length=30)

We would have to fix this path

File: README.md [hunk 2]
```diff
@@ -1,14 +1,731 @@
...
+  - Instructions for setting up WSL can be found [here](https://learn.microsoft.com/en-us/windows/wsl/install).
+  - To install PowerShell on WSL, follow the instructions [here](https://learn.microsoft.com/en-us/powershell/scripting/install/install-ubuntu?view=powershell-7.3).
+
+  To set the samples environment up, we will need the following tools to be installed prior to running the setup scripts.
+  1. Azure CLI version >= 2.57. Installation instructions [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux).
+  1. You need Docker for Linux installed locally. Installation instructions [here](https://docs.docker.com/engine/installation/#supported-platforms).
+  1. Confidential containers Azure CLI extension, version >= 0.3.5. You can install this extension using ```az extension add --name confcom -y```. You can check the version of the extension using ```az extension show --name confcom```. Learn about it [here](https://learn.microsoft.com/en-us/cli/azure/confcom?view=azure-cli-latest).
+  1. Managed CCF Azure CLI extension. You can install this extension using ```az extension add --name managedccfs -y```.
```
Comment (anantshankar17) [inline README.md:133]: >Managed CCF Azure CLI extension. You can install this extension using ```az extension add --name managedccfs -y```. [](http://example.com/codeflow?start=5&length=115)

Lets remove this :)

**Review by gaurav137 — APPROVED** [review]
:shipit:

**Comment by PhaniChiruvolu** [comment]
These changes are now a part of https://github.com/Azure-Samples/azure-cleanroom-samples

**Pull request closed without merging**